src/socket_type.cpp

```diff
@@ -279,7 +279,7 @@ namespace aux {
 		TORRENT_UNUSED(holder);
 		error_code e;
 		s.close(e);
-#endif // TORRENT_USE_OPENSSL
+#endif // TORRENT_SSL_PEERS
```

Comment:
this comment changed, but the corresponding `#if` didn't

Reply:
your are right